src/uproot/_dask.py

```diff
@@ -906,8 +902,14 @@ def __call__(self, file_path_object_path_istep_nsteps_ischunk):
     {ttree.object_path}
 
 which has {num_entries} entries"""
+                )
+        else:
+            events_per_step = math.ceil(num_entries / nsteps_or_stop)
```

Comment:
Here I added a `min` to ensure that we can't set `start` to `>num_entries`

Reply:
`start` can be equal to `num_entries`, and if so, the range will have length zero as you say.